README.md

```diff
@@ -1,37 +1,128 @@
+
+### Key repos and third-party components to focus on when onboarding
+
+#### Ruuter | Bükstack
+
+https://github.com/buerokratt/Ruuter
+
+All of our services are based on Ruuter. Whenever you want to make POST, GET or other requests, it's done via Ruuter. The same goes for if/else statements, creating and using templated services, etc. Ruuter also acts as a reverse proxy to mediate all traffic between any of the components. Go through the [guide](https://github.com/buerokratt/Ruuter/blob/main/samples/GUIDE.md) and definitely create your own [Ruuter-based mock services](https://github.com/buerokratt/Ruuter/blob/main/samples/steps/mock.md) to get the first grip of it.
+
+#### Resql | Bükstack
+
+https://github.com/buerokratt/Resql
+
+Resql is used for all of our [Postgres](https://www.postgresql.org/) requests. Every single database query lies in a separate `.sql` file and is consumable as a REST endpoint. You can take a look at [some of our chatbot-related database queries from here](https://github.com/buerokratt/Buerokratt-Chatbot/tree/main/DSL.Resql).
+
+#### DataMapper
+
+https://github.com/buerokratt/DataMapper
+
+DataMapper (you can also find it under the name of DMapper from here and there) is actually [https://handlebarsjs.com](https://handlebarsjs.com/) wrapped in Java. We will replace it with pure Handlebars on [Node.js](https://nodejs.org/en) soon. We use DataMapper to re-structure JSON outputs according to our needs. See some of our [currently in use DataMapper / Handlebars DSLs from here](https://github.com/buerokratt/Buerokratt-Chatbot/tree/main/DSL.DMapper).
+
+#### TIM | Bükstack
+
+https://github.com/buerokratt/TIM
+
+TIM is an acronym for "[TARA](https://www.ria.ee/en/state-information-system/electronic-identity-eid-and-trust-services/central-authentication-services) Integration Module", making it possible to start using TARA-based authentication by just updating your configuration file. TIM also provides functionality to create and use N number of custom [JWTs](https://en.wikipedia.org/wiki/JSON_Web_Token). PS! The source code of TIM is outdated and will be refactored in 2023 Q1/Q2.
+
+#### CVI
+
+https://github.com/buerokratt/cvi
+
+CVI is new to us but will be used a lot from now on. It's a fork of https://github.com/e-gov/cvi - also a very new initiative. Long story short - all of our GUI/front-end re-usable React componentes will end up there and will get committed back to the original project.
+
```

Comment:
https://toolbox.estonia.ee/pages/graphic-element - this is used regarding to widget